File: 2021년/SW Expert Academy/Python/2070.py
# 2개의 수를 입력 받아 크기를 비교하여 등호 또는 부등호를 출력하는 프로그램을 작성하라

test_case = int(input())

for i in range (1,test_case+1):
    first,second = map(int,input().split())
    if first < second:
        print(f"#{i} <")
    elif first == second:
        print(f"#{i} =")
    elif first > second:
        print(f"#{i} >")

File: 2021년/SW Expert Academy/C/2070.c
// 2개의 수를 입력 바아 크기를 비교하여 등호 또는 부등호를 출력하는 프로그램을 작성하라

#include <stdio.h>
#pragma warning (disable:4996)

main()
{
	int first, second, i, test_case;

	printf("Enter a test case :");
	scanf("%d", &test_case);

	for (i = 0; i < test_case; i++)
	{
		printf("Enter two numbers :");
		scanf("%d %d", &first, &second);

		if (first < second)
			printf("#%d <\n", i + 1);
		else if (first == second)
			printf("#%d =\n", i + 1);
		else
			printf("#%d >\n", i + 1);
	}
}